Gems/Atom/Feature/Common/Code/Source/PostProcessing/TaaPass.cpp

```diff
@@ -169,23 +176,30 @@ namespace AZ::Render
         Base::BuildInternal();
     }
 
-    void TaaPass::UpdateAttachmentImage(uint32_t attachmentIndex)
+    bool TaaPass::UpdateAttachmentImage(uint32_t attachmentIndex)
     {
         RPI::Ptr<RPI::PassAttachment>& attachment = m_accumulationAttachments[attachmentIndex];
         if (!attachment)
         {
-            return;
+            return false;
         }
 
         // update the image attachment descriptor to sync up size and format
         attachment->Update(true);
         RHI::ImageDescriptor& imageDesc = attachment->m_descriptor.m_image;
+
+        // The Format Source had no valid attachment
+        if (imageDesc.m_format == RHI::Format::Unknown)
+        {
+            return false;
+        }
+
         RPI::AttachmentImage* currentImage = azrtti_cast<RPI::AttachmentImage*>(attachment->m_importedResource.get());
 
         if (attachment->m_importedResource && imageDesc.m_size == currentImage->GetDescriptor().m_size)
         {
             // If there's a resource already and the size didn't change, just keep using the old AttachmentImage.
-            return;
+            return false;
```

Comment:
While this case shouldn't happen when `BuildInternal()` is called (only on `FrameBeginInternal()`), it seems odd to return false for this case since everything's really OK, it just doesn't need to update.

Reply:
changed to return true.